data/src/main/java/com/microsoft/azure/kusto/data/auth/HttpClientWrapper.java

```diff
@@ -135,34 +36,32 @@ private static boolean isNotContentLength(String name) {
 
     // Implementation of the synchronous HttpClient
     @Override
-    public IHttpResponse send(com.microsoft.aad.msal4j.HttpRequest httpRequest) throws Exception {
-        HttpUriRequest request;
-
-        // Translating the request
-
-        String uri = httpRequest.url().toString();
+    public IHttpResponse send(com.microsoft.aad.msal4j.HttpRequest httpRequest) {
```

Comment:
Do we still need the IHttpClient impl? what do we use it for?

Reply:
IHttpClient comes from MSAL4J. It sends HTTP requests that MSAL4J recognizes. In an upcoming PR I'd like to reduce it's necessity. We should probably get on a call about that though.